salt/orchestrate/aws/operations.sls

```diff
@@ -156,6 +186,52 @@ create_vault_security_group:
           to_port: 8200
           source_group_name: default
     - tags:
-        Name: vault-{{ VPC_RESOURCE_SUFFIX }}
+        Name: vault-{{ ENVIRONMENT }}
         business_unit: {{ BUSINESS_UNIT }}
         created_at: "{{ salt.status.time(format=ISO8601) }}"
+
+create_webapp_security_group:
+  boto_secgroup.present:
+    - name: webapp-{{ ENVIRONMENT }}
+    - vpc_name: {{ VPC_NAME }}
+    - description: ACL for web servers
+    - rules:
+        - ip_protocol: tcp
+          from_port: 80
+          to_port: 80
+          cidr_ip:
+            - 0.0.0.0/0
+            - '::/0'
+        - ip_protocol: tcp
+          from_port: 443
+          to_port: 443
+          cidr_ip:
+            - 0.0.0.0/0
+            - '::/0'
+    - tags:
+        Name: webapp-{{ ENVIRONMENT }}
+        business_unit: {{ BUSINESS_UNIT }}
+        Department: {{ BUSINESS_UNIT }}
+        OU: {{ BUSINESS_UNIT }}
+        Environment: {{ ENVIRONMENT }}
+
+create_webapp_odl_vpn_security_group:
```

Comment:
With this being added here, are you going to remove it from [here](https://github.com/mitodl/salt-ops/blob/master/salt/orchestrate/aws/security_groups.sls#L298)?